### ISSUE TITLE
count and for parameter limits

### ISSUE THREAD

**Issue by taggart** [sympa-community/sympa#287]
Today on our sympa install we noticed an interesting request in the webserver log:

`GET /www/rss/latest_lists?count=5000&for=5000 HTTP/1.1`

We has disabled requests to /www/latest_lists a while back because web crawlers kept triggering it and it was too cpu intensive for our large set of lists. But we didn't know about the rss feed.

Looking in wwsympa.fcgi there are a few different actions that can do rss with count= and for= parameters:

`active_lists, latest_arc, latest_d_read, latest_lists`

Would it make sense to have some sort of limit on count= and for= in order to limit the amount of work/output that a request can trigger?

**Comment by taggart** [comment]
We are still disabling this at the webserver level, but limits in sympa probably still make sense too as by default admins won't have that.

**Comment by ikedas** [comment]
I personally think the parameters for data size which can be manipulated from the outside to be the flaw of the system.  Such a thing should only be able to be manipulated through configuration by the administrator.

We'd appreciate if the fixes could be contributed.

**Comment by racke** [comment]
Yes, I agree there should be limit for count= and for= in the Sympa configuration.
